app/controllers/pages_controller.rb

```diff
@@ -7,7 +7,7 @@ def start
     session[:requested_path] = root_path
     if authenticated?
       if FeatureService.enabled?(:user_can_have_multiple_organisations) && organisation_not_set?
-        redirect_to(organisation_contexts_path) && return
+        redirect_to(organisations_path) && return
```

Comment:
this `organisation_contexts_path` didn't seem to exist in the routes file so I've changed it to `organisations_path`